Makefile

```diff
@@ -162,7 +168,16 @@ help:
 	@echo "         make TARGET WORKDIR=/tmp "
 	@echo ""
 	@echo "- REPO_URL"
-	@echo "use it to change the where rootfs base packages will be downloaded"
+	@echo "Use it to change the where rootfs base packages will be downloaded."
+	@echo ""
+	@echo "- IMG_SIZE"
+	@echo "Change the image size of the image to generate."
+	@echo ""
+	@echo "- DNF"
```

Comment:
it seems that could exits the case of `DNF=yum`, maybe could be better to rename that var, something like `PGK_MANAGER`